src/ape/api/accounts.py

```diff
@@ -135,9 +141,17 @@ def transfer(
             txn.data = self._convert(data, bytes)
 
         if value:
+            if "send_everything" in kwargs and kwargs["send_everything"]:
+                raise ValueError(
+                    "Kwarg send_everything=True requires transfer without value argument"
+                )
             txn.value = self._convert(value, int)
+            return self.call(txn, send_everything=value is None)
```

Comment:
since we know `send_everything` is False here
```suggestion
            return self.call(txn)
```